services/iam/doc/openapi.json

```diff
@@ -183,7 +188,14 @@
         },
         "responses": {
           "200": {
-            "description": "Success"
+            "description": "Success",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Tenant"
```

Comment:
As only the tenant id is being returned here, the schema should not define an entire Tenant, but should be updated to provide just an "id" property with string type.

Reply:
@weberjm I updated the schema. Let me know if it's good to go or if there are more changes to be made.